docs/basic-features/data-fetching.md

```diff
@@ -199,7 +199,11 @@ type Post = {
   content: string
 }
 
-export const getStaticProps = async () => {
+type StaticProps = {
+  posts: Post[]
+}
+
+export const getStaticProps<StaticProps> = async () => {
```

Comment:
As far as I can tell, this isn't needed because we're using `InferGetStaticPropsType` below 👍